dali/test/python/sequences_test_utils.py

```diff
@@ -220,59 +229,56 @@ def expand_arg(input_layout, num_expand, arg_has_frames, input_batch, arg_batch)
     return expanded_batch
 
 
-def expand_arg_input(input_data, input_layout, num_expand, arg_data, arg_has_frames):
-    assert(len(input_data) == len(arg_data))
-    return [expand_arg(input_layout, num_expand, arg_has_frames, input_batch, arg_batch)
-            for input_batch, arg_batch in zip(input_data, arg_data)]
-
+def expand_arg_input(input_data: ArgData, arg_data: ArgData):
```

Comment:
Is it replicating the `arg_data` if it's not per-frame in manner corresponding to the input data? I would not mind some small docstring here or there :P

Reply:
Yes, added the info about it in a docstr.